### DISCUSSION TITLE
Opening console in stride project.

### DISCUSSION THREAD

**Discussion by ghost** [stride3d/stride#1367] [Q&A]
i have a stride project that will act as a game server, i want to have a console open for admin purposes, however, calling Console.WriteLine(""), etc, doesn't open a console window. any suggestions for how i can have a console window open to interact with my project? debug solutions aren't an option. i'm fine with any solutions that require starting the game from the console though.

**Comment by manio143** [comment]
If you change the Windows project from WinExe to Exe (`<OutputType>Exe</OutputType>`) and run it from the terminal you'll see output of `Console.WriteLine`.
